www/css/main.diary.css

```diff
@@ -136,12 +142,24 @@ a.item-content {
   color: #333;
   font-size: 0.8em;
 } */
+
+
 .btn-input-confirm {
-  line-height: 30px;
+  width: 100%;
+  height: 30px;
+  margin: 2px;
+  box-sizing: border-box;
+  border-radius: 10px;
+  font-family: 'Lato';
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
```

Comment:
I believe that I had pulled this change from the Safari debugger because it made the design more appealing. I don't remember this specific change, but I don't mind switching it back.

Reply:
I will make an issue to complete this for now, it looks like this change wasn't completed. It looks the same too; here is the `font-size` changed back to `0.8em`
![Simulator Screen Shot - iPhone 11 - 2022-08-31 at 11 59 16](https://user-images.githubusercontent.com/61334340/187749715-5d92c7b1-4845-437e-9c6f-af3fc7b8dfa3.png)

Reply:
ah in this one, the font looks like it is the same as the dashboard. If this screenshot includes all your changes, this is good to go.

Reply:
> ah in this one, the font looks like it is the same as the dashboard. If this screenshot includes all your changes, this is good to go.

Acknowledged. **_Confirmed this change as complete_**